eng/package.ps1

```diff
@@ -9,9 +12,27 @@ $csproj = Join-Path $PSScriptRoot ../src/DotNet/DotNet.csproj
 $ext = if ($IsWindows) { ".exe" } else { "" }
 $dotnet = Join-Path $PSScriptRoot ../bin/dotnet/dotnet$ext
 $sln = Join-Path $PSScriptRoot ../Microsoft.Maui-net6.sln
+$winuisln = Join-Path $PSScriptRoot ../Microsoft.Maui.WinUI.sln
 
+# Build with .\bin\dotnet\dotnet.exe
+& $dotnet build $sln `
+    -c:$configuration `
+    -p:SymbolPackageFormat=snupkg `
+    -bl:$artifacts/maui-build-$configuration.binlog
+
+if ($IsWindows)
+{
+  & $msbuild $winuisln `
+    /p:configuration=$configuration `
+    /p:SymbolPackageFormat=snupkg `
+    /t:restore;build
```

Comment:
Can you change this to set `-p:MauiFrameworks`, so this command would only build the WinUI parts?